src/environment.ts

```diff
@@ -19,14 +19,14 @@ function installLazyGlobals(env: SecureEnvironment, baseGlobalThis: object, base
             // For now, we just do a manual composition:
             const { set: originalSetter, get: originalGetter, value: originalValue } = descriptor;
             if (!isUndefined(originalGetter)) {
-                descriptor.get = function get(): any {
-                    const value: any = apply(originalGetter, baseGlobalThis, []);
+                descriptor.get = function get(): SecureValue {
+                    const value: RawValue = apply(originalGetter, baseGlobalThis, emptyArray);
                     return env.getSecureValue(value);
                 };
             }
             if (!isUndefined(originalSetter)) {
-                descriptor.set = function set(v: any): void {
-                    apply(originalSetter, baseGlobalThis, [v]);
+                descriptor.set = function set(v: SecureValue): void {
+                    apply(originalSetter, baseGlobalThis, [env.getRawValue(v)]);
```

Comment:
@jdalton this is the relevant bug... if a global setter is installed inside the sandbox, what should happen when the sandbox code is attempting to mutate that? I believe it should only change the sandbox's global object rather than mutating the outer realm's global reference instead of attempting to unwrap and set the unwrapped value... that seems odd because it violates the first rule: never let the sandbox to change the obj graph of the outer realm. I need to think more about this.